src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -495,6 +495,14 @@
     <Field type="uint" name="gcId" label="GC Identifier" relation="GcId" />
   </Event>
 
+  <Event name="GCCpuTime" category="Java Virtual Machine, GC, Detailed" label="GC CPU Time" description="GC CPU Time information in seconds"
```

Comment:
Could you change the name to "GCCPUTime". Other events uses capital letters in abbreviations.

Reply:
Renamed to "GCCPUTime".